# Admittance parameters for mass damper systems 

mass_matrix: [2,0,0,0,0,0,
              0,2,0,0,0,0,
              0,0,2,0,0,0,
              0,0,0,1,0,0,
              0,0,0,0,1,0,
              0,0,0,0,0,1]

damping_matrix: [40,0,0,0,0,0,
                 0,40,0,0,0,0,
                 0,0,40,0,0,0,
                 0,0,0,10,0,0,
                 0,0,0,0,10,0,
                 0,0,0,0,0,10]

# this will limit the workspace ([x_min x_max y_min y_max z_min z_max])
workspace_limits: [0.45, 1.5, -1.3, 1.3, 0.10, 1.4]

# joint limit equal to 2*pi
joint_limits: [6.283, 6.283, 6.283, 6.283, 6.283, 6.283]

# kinematics manipulator limits
maximum_velocity: [1.05, 1.05, 1.57, 1.57, 1.57, 1.57]
# maximum_acceleration: [0.57, 0.57, 0.57, 0.57, 0.57, 0.57]
maximum_acceleration: [0.20, 0.20, 0.20, 0.20, 0.20, 0.20]

# force sensor dead zone
force_dead_zone: 4.0
torque_dead_zone: 1

# admittance control weight
admittance_weight: 2

# inertia reduction bool
inertia_reduction: false
